src/pages/ProjectSettings.tsx

```diff
@@ -68,7 +68,7 @@ export const ProjectSettings: React.FC = () => {
     variables: { repoId: repoRefId },
     onError: (e) => {
       dispatchToast.error(
-        `There was an error loading the repo ${repoRefId}: ${e.message}`
```

Comment:
Will the message always contain a helpful enough error? It seems like the "Error!" already indicates there was an error and just printing e.message would be sufficient

Reply:
I think by convention we wouldn't want to rely on just e.message since it's out of Spruce's control and may change. We'd want to wrap it in our own error explanation.

Reply:
I'd prefer to shorten it since I do think the title covers it but I'll defer to UI conventions; cc @khelif96 if you can weigh in